lib/hanami/commands/server.rb

```diff
@@ -24,12 +27,15 @@ class Server < ::Rack::Server
       # @since 0.1.0
       # @see Hanami::Environment#initialize
       def initialize(options)
+        @entr_options =  options.merge(code_reloading: false).inject([]) {|res, (k, v)| res << "--#{k}=#{v}" ; res}.join(" ")
         @_env    = Hanami::Environment.new(options)
         @options = _extract_options(@_env)
-
         if code_reloading?
+          begin
```

Comment:
@aderyabin Can we please declutter `#initialize`, by extracting a method like `prepare_code_reloading!`? Thank you! :+1: This should include `@entr_options` line too. :smile: